### ISSUE TITLE
No English sentence is provided in CC Matrix data and some language pairs are missing.

### ISSUE THREAD

**Issue by TomokiMatsuno** [facebookresearch/LASER#175]
Hi

I've downloaded files listed in `https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0.beta/list.txt` and extracted the entries in the language information column (e.g., cs-ko/ko) in there.

Then, I've found that there is no English sentence (i.e., "bla-en/en" or "en-bla/en").

Furthermore, some popular language pairs like es-pt (Spanish and Portuguese) are missing.

Is there something wrong or these pairs are not provided intentionally?

**Comment by TomokiMatsuno** [comment]
Here are the commands I used for the experiment.

```
wget https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0.beta/list.txt
cat list.txt | xargs -I{} curl -s https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0.beta/{} | zcat | cut -f1 > langs.txt
sort --parallel $(nproc) langs.txt | uniq -c | sort -rn > lang_stats.txt
grep "/en" lang_stats.txt 
```

**Comment by fastandslow007** [comment]
+1.
Is it really removed for a reason? 
To achieve no English-centric data; [https://ai.facebook.com/blog/introducing-many-to-many-multilingual-machine-translation/](https://ai.facebook.com/blog/introducing-many-to-many-multilingual-machine-translation/)

**Comment by TomokiMatsuno** [comment]
Actually, many language pairs without English are missing.

And most of language pairs have much fewer sentences than the numbers described in the paper, and the number of sentences in each direction is not equal.

For example, according to `lang_stats.txt` in https://github.com/facebookresearch/LASER/issues/175#issuecomment-768816498, "ja-pt/ja" (Japanese part of Japanese-Portuguese) has 887 sentences and "ja-pt/ja" (Portuguese part of Japanese-Portuguese) has 276 although the [CCMatrix paper](https://arxiv.org/abs/1911.04944) says there are 9.1 million sentences in the pair.

**Comment by gwenzek** [comment]
We had issues with the japanese bitexts, and they aren't part of the beta release. English was also to big to be part of the beta release.
I've made progress on the full release, and I'll update the issues soon.
The numbers won't exactly match the numbers from the CCMatrix paper, their was some losses in the reverse processing of the corpus.

**Comment by gwenzek** [comment]
Hi all, we had some issues with the release of the missing dataset. I wasn't able to find the bitext for the following languages:
am, ast, cy, fy, ga, gd, ha, ig, jv, km, lb, lg, my, om, or, so, wo, xh, yi, yo, zu.

I'll still continue trying to recover them but in the meantime I added v1.0.0 on S3 which should have the rest of the languages.
My validation script hasn't finished running , so I hope I'm not creating false hope here. But since I'm away next week and the data take a long time to download anyway, I figured I should share it.

Thanks for your patience and sorry for the delay.
